sql/updates/world/2016_01_23_XX_world.sql

```diff
@@ -0,0 +1,32 @@
+/* Sunken Temple - Support for quest 3447: Secret of the Circle */
+SET @GO_GUID := 9999999;
+SET @ALTAR := 148836;
+SET @LIGHT := 148883;
+SET @ATAL_ALARION := 8580;
```

Comment:
SQL files usually do not need text alignment like in the .cpp files,
but it is common practice to align a "set @ variables" section:
```sql
SET @GO_GUID      := 9999999;
SET @ALTAR        :=  148836;
SET @LIGHT        :=  148883;
SET @ATAL_ALARION :=    8580;
SET @ATALAI_IDOL  :=  148838;
```
The order of these variables can be sorted either by variable name, order of appearance in the SQL script or by value, but it is not important and usually chosen by the author. The GUID value to be set by TC is usually first, just like you have set it here.